02-GitHub.Rmd

```diff
@@ -9,67 +9,74 @@ opts_chunk$set(tidy.opts=list(width.cutoff=50), tidy = FALSE)
 
 Trying to find instructions on how to use git and GitHub online is notoriously difficult for beginners. You'll often come across instructions like this:
 
-> Doing X is easy. All you have to do is open up a terminal and type git -s -x -what -thehell -doesthisallevenmean
+> Doing X is easy. All you have to do is open up a terminal and type 
+>
+> `git -s -x -what -thehell -doesthisallevenmean`
 
-Thankfully, using git and GitHub is super easy and you don't have to use the command line at all to benefit from these tools.
+Thankfully, using git and GitHub doesn't have to be this hard!
+You don't even have to use the command line at all to benefit from these tools.
 
 To get started, it is important to know what git and GitHub are and how they differ.
 
 Image credit: devmountian.com
 
 ## Git {#Git2.1}
 
-*Git* is software that (usually) runs on your computer. It is used to tracks all changes within a particular folder. Any change within that folder --- a deleted file, a new line in a file, or a single changed letter in a single file --- is tracked. It's as if Microsoft word's "Track Changes" function donned a cape, started solving crime, and kept meticulous notes all the while.
+*Git* is software that (usually) runs on your computer. It is used to track all changes within a particular folder. Any change within that folder --- a deleted file, a new line in a file, or a single changed letter in a single file --- is tracked. It's as if Microsoft word's "Track Changes" function donned a cape, started solving crime, and kept meticulous notes all the while.
 
-A folder on your computer that is tracked by .git is called a `repository` or `repo` for short. The main difference between a .git repo and a regular folder on your computer is that a git repo has a folder within it named .git. It's within this .git folder that all the changes in the main folder are tracked. Other than that, a git repo is just a folder on your computer like any other, but with new and wonderful powers.
```

Comment:
I think you should refer to the actual `.git` file as .git only. The software should be called Git, and perhaps the actual program (where you talk about downloading it) should be called `git`, only because that's the terminal command for it.